### PR TITLE
Config/4 docker image

### PR DESCRIPTION
**Pull request by AntonioRodriguezRuiz**
## Description
Added image metadata and fixed databse connection command not working by using mongoose

### CODE REVIEW

**Comment by AntonioRodriguezRuiz** [comment]
## Changes
copilot:walkthrough

File: .github/workflows/docker-deploy.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ on:
       - labeled
 
 env:
-  REGISTRY_IMAGE: ${{ secrets.DOCKER_USERNAME }}/AQUÍ_VA_EL_NOMBRE_DE_TU_IMAGEN_SIN_MAYUSCULAS
-  GHCR_IMAGE: fis2425/AQUÍ_VA_EL_NOMBRE_DE_TU_IMAGEN_SIN_MAYUSCULAS
+  REGISTRY_IMAGE: ${{ secrets.DOCKER_USERNAME }}/authorization-svc
```
Comment (alvarobernal2412) [inline .github/workflows/docker-deploy.yml:16]: To follow the same naming strategy, I recommend using only the name of the service (Example: `authorization`). You can check existing images [here](https://hub.docker.com/repositories/cloudmedix)

**Review by alvarobernal2412 — CHANGES_REQUESTED** [review]
Almost there!

File: .github/workflows/docker-deploy.yml (hunk 1 above)
Comment (AntonioRodriguezRuiz) [reply .github/workflows/docker-deploy.yml:16]: that makes sense. Will do

Comment (AntonioRodriguezRuiz) [reply .github/workflows/docker-deploy.yml:16]: Do you want me to amend the previous commit or create a new one?

**Pull request merged**